packages/vats/src/core/utils.js

```diff
@@ -63,6 +66,52 @@ export const feeIssuerConfig = {
   displayInfo: { decimalPlaces: 6, assetKind: AssetKind.NAT },
 };
 
+/**
+ * @template {AssetKind} K
+ *
+ * Create a brand that cannot ever produce mints, issuers, payments or purses.
+ *
+ * The allegedName becomes part of the brand in asset descriptions. The
+ * allegedName doesn't have to be a string, but it will only be used for
+ * its value. The allegedName is useful for debugging and double-checking
+ * assumptions, but should not be trusted.
```

Comment:
"trusted" is kinda fuzzy... I tend to phrase this as: no code should depend on an allegedName for correctness. MarkM tends to emphasize that outside of debug consoles, an allegedName is never shown to a user.

OTOH... there's nothing novel about the use of allegedName here vs. any other ERTP usage. Perhaps it's more straightforward to leave this comment out?